packages/clerk-js/src/core/resources/Organization.ts

```diff
@@ -342,18 +342,16 @@ export class Organization extends BaseResource implements OrganizationResource {
 
   public async reload(params?: ClerkResourceReloadParams): Promise<this> {
     const { rotatingTokenNonce } = params || {};
-    const json = await BaseResource._fetch(
-      {
+
+    const json = (
+      await BaseResource._fetch<OrganizationJSON>({
+        path: `/organizations/${this.id}`,
         method: 'GET',
-        path: `/me/organization_memberships`,
         rotatingTokenNonce,
-      },
-      { forceUpdateClient: true },
```

Comment:
Don't we need to include this parameter as well?

Reply:
brought it back